main.py

```diff
@@ -63,7 +63,8 @@ def login():
         # get unit number/email field
         units = request.form.get("units")
         if units is None:
-            return index()
+            flash(u'Please enter an email/unit number.', 'error')
```

Comment:
The `u` prefix is used to explicitly state that the string is `utf-8` encoded. In Python 3, this is default. No need for this here.